bin/resources/GameIndex.yaml

```diff
@@ -16140,6 +16205,13 @@ SLES-52573:
   region: "PAL-M5"
   roundModes:
     eeRoundMode: 2 # Fixes abnormal character behaviour.
+SLED-52574:
```

Comment:
why did you move this? it was in the right place, it's SLED not SLES

Reply:
Old local file